clib/cArray.mli

```diff
@@ -114,6 +114,10 @@ sig
   val fold_left2_map : ('a -> 'b -> 'c -> 'a * 'd) -> 'a -> 'b array -> 'c array -> 'a * 'd array
   (** Same with two arrays, folding on the left; see also [Smart.fold_left2_map] *)
 
+  val fold_left2_map_i :
+    (int -> 'a -> 'b -> 'c -> 'a * 'd) -> 'a -> 'b array -> 'c array -> 'a * 'd array
+  (** Same than [fold_left2_map_i] but passing the index of the array *)
```

Comment:
Did you mean `Same as [fold_left2_map]`?

Reply:
Ups, fixed thanks!